### PR TITLE
Update UI doc for connecting to S3

### PR DESCRIPTION
**Pull request by likawind**
This PR mainly update UI doc for S3 connection by:
* update instructions and screenshots for upload file and specify file path
* remove enterprise section and community section title

### CODE REVIEW

File: integrations/adding-an-integration/connecting-to-aws-s3.md [hunk 1]
```diff
@@ -1,63 +1,24 @@
...
+Ensure that your IAM user has access to the S3 Bucket you're connecting to Aqueduct. There are 3 ways to provide your credentials to Aqueduct.
 
-Ensure that your IAM user has access to the S3 Bucket you're connecting to Aqueduct. You will need the Access Key ID and the Secret Access Key. Enter the name of the bucket, the Key ID, and the Key into the S3 connection form on the Aqueduct UI and click Connect:
+## Manually Enter Credentials
+You can manually copy over the Key ID and the Secret Key.
+![](<../../.gitbook/assets/integration_s3_credential_manual.png>)
 
-![](<../../.gitbook/assets/connect_aws.png>)
+## Upload Your Credentials File
+If you are using [AWS cli](https://aws.amazon.com/cli/) or other AWS SDK on your machine, you might have your credentials file available to you, typically under `<HOME>/.aws/credentials`. You can upload this file to Aqueduct and specify the profile name you would like to use.
```
Comment (eunice-chan) [inline integrations/adding-an-integration/connecting-to-aws-s3.md:12]: Nit: AWS cli --> AWS CLI

File: integrations/adding-an-integration/connecting-to-aws-s3.md [hunk 2]
```diff
@@ -1,63 +1,24 @@
...
+![](<../../.gitbook/assets/integration_s3_credential_upload.png>)
 
+## Specify the Path to Your Credentials File
+### Access Key and Secret Access Key
+If you are using [AWS cli](https://aws.amazon.com/cli/) or other AWS SDK on your machine, you might have your credentials file available to you, typically under `<HOME>/.aws/credentials`. If your server is running on the same machine as this file, you can specify the **absolute path** to this file together with the profile name you would like to use. When you provide credentials by file path, any updates to the file contents will **automatically update all workflows** using this integration.
```
Comment (eunice-chan) [inline integrations/adding-an-integration/connecting-to-aws-s3.md:17]: The first sentence is a duplicate of the above section's first sentence. Maybe we can consolidate the sections e.g. `Using Credential Files` to only have it once and put Upload vs Specify as subsections.

**Review by eunice-chan — COMMENTED** [review]
@vsreekanti  Not sure if we still want to have references to Community vs Enterprise or if we would want to add that in once we have an Enterprise product launched.

**Comment by likawind** [comment]
Thanks @eunice-chan , updated for most comments. The only thing I'm a bit hesitate is the structure, but I tried to made some edit so that it better follows the user's mind set. The current structure looks like this:
* Summary
  * Access Key
    * manual enter
    * path
    * upload
  * SSO (path only)
 My idea is to let user first decide what credential to use and then decide how to provide them. In this way they can just click into each link and get fully explained on how to provide that credentials. Despite some copying, I feel the 'final' section is more self-contained and user don't need to jump between sections to get full context

File: integrations/adding-an-integration/connecting-to-aws-s3.md [hunk 3]
```diff
@@ -1,63 +1,25 @@
...
 
-![](<../../.gitbook/assets/connect_aws.png>)
+## AWS SSO
+Aqueduct can connect to S3 using your [AWS SSO](https://aws.amazon.com/iam/identity-center/) credentials. For now, specifying the path is **the only way** to use these credentials. This also means your server **must** be running on the same machine as the one with SSO access.&#x20;
 
+When using [AWS SSO](https://aws.amazon.com/iam/identity-center/), your credentials file is typically be under `<HOME>/.aws/config` (this is **different** from `<HOME>/.aws/credentials`). To connect to S3, you can specify the **absolute path** to this file together with the profile name you would like to use. Once the path is specified, any updates to the file contents (for example, [re-authenticate SSO using browser](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sso.html)) will **automatically update all workflows** using this integration.&#x20;
 
+![](<../../.gitbook/assets/integration_s3_credential_path.png>)
 
-## Enterprise Edition
```
Comment (vsreekanti) [inline integrations/adding-an-integration/connecting-to-aws-s3.md:13]: I'm good with removing this for now.

**Review by vsreekanti — APPROVED** [review]
@likawind, this looks good! There are a few minor copy edits, but it might be faster for us to merge and then for me to make those changes on Gitbook directly. Feel free to merge whenever. 🙂

**Pull request merged**